Explorer/Assets/DCL/Roads/Systems/RoadInstantiatorSystem.cs

```diff
@@ -50,15 +51,17 @@ protected override void Update(float t)
         [None(typeof(DeleteEntityIntention))]
         private void InstantiateRoad(ref RoadInfo roadInfo, ref SceneDefinitionComponent sceneDefinitionComponent, ref PartitionComponent partitionComponent)
         {
+            if (sceneDefinitionComponent.IsPortableExperience) return;
```

Comment:
Would it work if the query has: `[None(typeof(PortableExperienceComponent))]` ?

Reply:
TBH, as it needs the RoadInfo component and for sure PX wont ever have it, it should not need that check at all, but adding the None at the top as you suggest shouldnt hurt either, thank you for spotting it!!